experiment/build/generate_cloudbuild.py

```diff
@@ -13,28 +13,69 @@
 # limitations under the License.
 """Generates Cloud Build specification"""
 
-import argparse
-import json
-import yaml
+import os
 
 from experiment.build import docker_images
 
+BASE_TAG = 'gcr.io/fuzzbench'
+EXPERIMENT_TAG = "${_REPO}"
+FUZZER_VAR = "${_FUZZER}"
+BENCHMARK_VAR = "${_BENCHMARK}"
+EXPERIMENT_VAR = "${_EXPERIMENT}"
 
-# TODO(Tanq16): Add unit test for this.
-def create_cloud_build_spec(buildable_images, docker_registry):
-    """Returns Cloud Build specificatiion."""
+
+def _identity(name):
+    return name.replace(FUZZER_VAR, "fuzzer").replace(BENCHMARK_VAR,
+                                                      "benchmark")
+
+
+def coverage_steps():
+    """Return gcb run steps for coverage."""
```

Comment:
`Returns`, dosctrings should be descriptive style not imperative. See https://google.github.io/styleguide/pyguide.html#383-functions-and-methods